Gemfile

```diff
@@ -78,7 +78,7 @@ gem 'pundit'
 gem 'toastr-rails', '~> 1.0', '>= 1.0.3'
 
 
-gem 'rails-assets-sweetalert2', source: 'https://rails-assets.org'
+gem 'rails-assets-sweetalert2', source: 'http://insecure.rails-assets.org'
```

Comment:
Why do we need to make the connection insecure?

Reply:
When I `bundle install` with out this change, it's throwing an error. I found the solution [here](https://github.com/tenex/rails-assets/issues/325#issuecomment-223044836).

And the full log of the error is:
```
Fetching source index from https://rails-assets.org/

Retrying fetcher due to error (2/4): Bundler::Fetcher::CertificateFailureError Could not verify the SSL certificate for https://rails-assets.org/.
There is a chance you are experiencing a man-in-the-middle attack, but most likely your system doesn't have the CA certificates needed for verification. For information about OpenSSL certificates, see http://bit.ly/ruby-ssl. To connect without using SSL, edit your Gemfile sources and change 'https' to 'http'.
Retrying fetcher due to error (3/4): Bundler::Fetcher::CertificateFailureError Could not verify the SSL certificate for https://rails-assets.org/.
There is a chance you are experiencing a man-in-the-middle attack, but most likely your system doesn't have the CA certificates needed for verification. For information about OpenSSL certificates, see http://bit.ly/ruby-ssl. To connect without using SSL, edit your Gemfile sources and change 'https' to 'http'.
Retrying fetcher due to error (4/4): Bundler::Fetcher::CertificateFailureError Could not verify the SSL certificate for https://rails-assets.org/.
There is a chance you are experiencing a man-in-the-middle attack, but most likely your system doesn't have the CA certificates needed for verification. For information about OpenSSL certificates, see http://bit.ly/ruby-ssl. To connect without using SSL, edit your Gemfile sources and change 'https' to 'http'.
Could not verify the SSL certificate for https://rails-assets.org/.
There is a chance you are experiencing a man-in-the-middle attack, but most
likely your system doesn't have the CA certificates needed for verification. For
information about OpenSSL certificates, see http://bit.ly/ruby-ssl. To connect
without using SSL, edit your Gemfile sources and change 'https' to 'http'.
```

Reply:
@nathnael I fixed this and I have reverted back to secure setting. I am pushing that change now. For reference I will put the link that helped me resolve it here.

http://railsapps.github.io/openssl-certificate-verify-failed.html